### ISSUE TITLE
Media Upload on Repeater

### ISSUE THREAD

**Issue by andr3ribeiro** [moyus/easy-options-framework#1]
Hi!
First, congratulations for the great Idea and well coded framework! It's nice!

Just one thing, I tried to insert Media Upload inside the repeater field. The modal with image upload doesn't appear. With same field outsite Repeater, the modal appears properly.

Is there anything else I need to do to Media Field work inside the Repeater?

Tks a lot!

**Comment by moyus** [comment]
Haven't update this plugin for a long time, maybe it's not compatible with latest WordPress version. I will take a look and fix it soon.

**Comment by moyus** [comment]
I have tested it in WordPress 5.5.3, it worked. How do you setup your configs, do you follow the `sample-config.php`?

**Comment by andr3ribeiro** [comment]
Hi,

Yes, using the sample file.
When I use upload media as a single field, works properly!
When I use inside a repeater, the window to choose a file doesn't appear.

**Comment by moyus** [comment]
Can you attach your config file?

**Comment by andr3ribeiro** [comment]
Hi,
I was using your framework to solve one question for a client. I already solved it without the framework.

Now I downloaded the project again and start a fresh new configuration from scratch... And all is working!
Last time I do something wrong with the config file. My mistake.

I have another different question about:
Can I have control of media upload thumbnail size? Large images are showing with max-width:100%; on the admin page.

That is the reference:
https://github.com/moyus/easy-options-framework/blob/8e00bf9404a0784fadcd71018c22a3f11f327ae9/fields/media.php#L98

Maybe not, because the CSS rule is hard-coded here, right?
